src/Twig/Extension/TemplateExtension.php

```diff
@@ -37,7 +37,7 @@ class TemplateExtension extends AbstractExtension
     /**
      * @param bool $debug Is Symfony debug enabled?
      */
-    public function __construct($debug, TranslatorInterface $translator, AdapterInterface $modelAdapter)
+    public function __construct($debug, TranslatorInterface $translator, AdapterChain $modelAdapter)
```

Comment:
In completion :wink: 

```suggestion
    public function __construct($debug, TranslatorInterface $translator, AdapterInterface $modelAdapter)
```